mne/io/edf/edf.py

```diff
@@ -1020,6 +1028,11 @@ def _read_gdf_header(fname, stim_channel, exclude):
                                 for i, t in enumerate(dtype)])
 
             # Populate edf_info
+            if date == DATE_NONE:
+                _meas_date = DATE_NONE
+            else:
+                _meas_date = (calendar.timegm(date.utctimetuple()), 0)
```

Comment:
FYI I have a more complete fix for this on the way

Reply:
Do you want me to revert it?